user_manual/files/deletedfiles.rst

```diff
@@ -1,21 +1,59 @@
-Deleted Files
-=============
-
-ownCloud keeps a copy of your deleted files in case you need them again. To
-make sure that the user doesn't run out of memory, the deleted files app
-manages the size of the deleted files for the user. The app takes care to never
-use more that 50% of your currently available free space. If your deleted files
-exceed this limit, ownCloud deletes the oldest versions until it meets the memory
-usage limit again.
-
-Deleted files can be found by clicking on the *Deleted files* button on files app of web interface.
-You can either restore or permanently delete using appropriate buttons.
-
-ownCloud also checks the age of deleted files every time new files are added
-to the deleted files. By default, deleted files stay in the trash bin for 180 days.
-The Administrator can adjust this value in the config.php by setting the
-***"trashbin_retention_obligation"*** value. Files older than the 
-***"trashbin_retention_obligation"*** will be deleted permanently.
-Additionally, ownCloud calculates the maximum available space every time
-a new file is added. If the deleted files exceed the new maximum allowed space
-ownCloud will expire old deleted files until the limit is met once again.
+======================
+Managing Deleted Files
+======================
+
+When you delete a file in ownCloud, it is not immediately deleted permanently. 
+Instead, it is moved into the trash bin. It is not permanently deleted until 
+you manually delete it, or when the Deleted Files app deletes it to make room 
+for new files.
+
+Find your deleted files by clicking on the **Deleted files** 
+button on the Files page of the ownCloud Web interface. You'll have options to 
+either restore or permanently delete files.
+
+Quotas
+------
+
+Deleted files are not counted against your storage quota. Only files that 
+originate with users count against their quotas, and not files 
+shared with them that originate from other users. (See :doc:`quota` to learn 
+more about quotas.)
+
+What Happens When Shared Files Are Deleted
+------------------------------------------
+
+Deleting files gets a little complicated when they are shared files, and when 
+you delete a shared file it may not delete copies held by other users, as this 
+scenario illustrates:
+
+1. User1 shares a folder "test" with User2 and User3
+2. User2 (the recipient) deletes a file/folder "sub" inside of "test"
+3. The folder "sub" will be moved to the trashbin of both User1 (owner) and 
+   User2 (recipient)
+4. But User3 will not have a copy of "sub" in her trash bin
+
+When User1 deletes "sub" then it is moved to User1's trashbin, and it is not 
+deleted from User2 and User3.
```

Comment:
well spotted, the file will be also deleted for user2 and user3 but it will be only in the trashbin for of user1